x-pack/lib/monitoring/inputs/metrics/state_event_factory.rb

```diff
@@ -29,7 +29,7 @@ def pipeline_data(pipeline)
         "ephemeral_id" => pipeline.ephemeral_id,
         "workers" =>  pipeline.settings.get("pipeline.workers"),
         "batch_size" =>  pipeline.settings.get("pipeline.batch.size"),
-        "representation" => ::LogStash::Inputs::Metrics::StateEvent::LIRSerializer.serialize(pipeline.lir)
+        "representation" => ::LogStash::Inputs::Metrics::StateEvent::XPackLIRSerializer.serialize(pipeline.lir)
```

Comment:
this should be able to use the serialiser that was moved OSS-code directly, and shouldn't need to access it through an Elastic-Licensed subclass that is functionally identical.